SpatialGDK/Source/SpatialGDK/Public/EngineClasses/Components/SpatialPingComponent.h

```diff
@@ -39,9 +39,14 @@ class SPATIALGDK_API USpatialPingComponent : public UActorComponent
 
 private:
 	float RoundTripPing;
-	float LastReceivedPingTimestamp;
-	UPROPERTY(ReplicatedUsing = OnRep_ReplicatedPingTimestamp)
-	float ReplicatedPingTimestamp;
+
+	TArray<int16> SentPingIDs;
```

Comment:
Can we instead have one array that contains structs with both the ID and the timestamp?
Also, if the intention is that the IDs will wrap around, we should use `uint16` as signed wrap around is undefined according to the standard (although it will work as expected on most platforms, we shouldn't rely on that).

Reply:
I'm actually just about to push some changes that replace that array structure entirely, sorry. I've opted for something closer to the standard ping pong model with a timeout. Saves storing anything besides the last sent ping.

Reply:
I've switched to uint16 for the ID though.